lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -71,6 +71,9 @@ def build_index_from_metadata(self) -> str:
             Retrieve the list of ES indices from the dataset index
             map metadata based on a given root index name.
 
+            Args:
+                root_index_name: root index name
+
```

Comment:
This function doesn't have any arguments (other than the `self` reference)...did you paste this in the wrong place?

Reply:
There was a PR posted by nikhil it was a change before that. I will remove it.